starter-code/scripts/articleView.js

```diff
@@ -75,7 +75,13 @@ articleView.setTeasers = function() {
 
 articleView.initNewArticlePage = function() {
  // TODO: Make the tabs work. Right now, you're seeing all the tab content (items with a class of tab-content) on the page at once. The section with the id of "write" should show when the "write" tab is clicked; it is also the default and should be shown on page load. The section with the id of "articles" should show when the "preview" tab is clicked.
+  $('.tab').on('click', '[data-content="write"]', function(){
```

Comment:
This doesn't seem to be working right. Nothing appears to happen when you click on the tabs